product/product.go

```diff
@@ -313,15 +313,15 @@ func (p *APIProduct) authorize(authContext *auth.Context, api, path, method stri
 			var valid bool
 			valid, hint = oc.isValidOperation(api, path, method, hints)
 			if valid {
-				// api is already defined outside this block as a string so ao is used here to avoid confusion
 				ao := AuthorizedOperation{
-					ID:            fmt.Sprintf("%s-%s-%s-%s", p.Name, env, authContext.Application, oc.ID),
+					ID:            fmt.Sprintf("%s-%s-%s-%s", p.Name, env, authContext.Application, oc.APISource),
```

Comment:
This ID is still different from the one on line 352, IIUC that's not sharing the same product-level quota bucket?

Reply:
Thank you! Yes, it absolutely should be the same.